src/app/app.js

```diff
@@ -1,8 +1,12 @@
+"use strict";
+
 import Hello from "./components/hello.jsx";
```

Comment:
Let's remove unused import statements.  I think we can get rid of these components now that we have some real functionality. @yshen01